### DISCUSSION TITLE
Screen lock

### DISCUSSION THREAD

**Discussion by GuG1050** [devemux86/cruiser#445] [Q&A] [answered]
I was searching for the function "display lock" (Displaysperre) wich existed in the past to prevent rain from operating the phone. Has this feature been eliminated? I cant find it anymore ... Version 3.0.40

**Comment by devemux86** [comment]
@GuG1050 

It is now part of the new quick actions in navigation / follow. Press the compass to use them.

- https://github.com/devemux86/cruiser/discussions/405

<img src="https://github.com/devemux86/cruiser/assets/3484020/e6ec7fc9-7145-45b8-89a3-0850fb9b7482" width="50%"/>

**Comment by GuG1050** [comment, marked as answer]
Thanks. Failed to find it because no Routing active ...

**Reply by bios4** [reply]
> Failed to find it because no Routing active ...

Which is a VERY good point!

How to activate the display lock during "follow mode" @devemux86???

It might be helpful also when driving in rain w/o active routing, so I would appreciate to have this available also then...

**Reply by devemux86** [reply]
@bios4 thanks for the report.

Quick actions (tap compass) will also work in follow mode in the next update.

**Reply by bios4** [reply]
> Quick actions (tap compass) will also work in follow mode in the next update.

Just to be sure: this is not yet implemented in the current 3.0.41 Beta 1, correct?

**Reply by devemux86** [reply]
@bios4 It should also work in follow mode in:
- https://github.com/devemux86/cruiser/discussions/451

**Reply by bios4** [reply]
> @bios4 It should also work in follow mode in:
> 
>     * [Cruiser 3.0.41 (Beta) #451](https://github.com/devemux86/cruiser/discussions/451)

Long-press on compass only gives me the following selection in 3.0.41 Beta:

<img alt="" src="https://github.com/devemux86/cruiser/assets/145460147/57ceade1-f79a-4db0-a33e-d514f94e2ef8" width="33%">

Am I doing something wrong?

EDIT: solved it - long-press <> tap... Sorry for the inconvenience!

**Reply by devemux86** [reply]
@bios4 

- Tap for actions
- Long press for options

Everywhere in the app there is this rule, also usually common in Android.

**Comment by bios4** [comment]
I found the screen lock option to be a very useful function during my recent motorcycle holidays in Gran Canaria, as it successfully prevented unintentional inputs during rain...

However, having to disable the screen lock via the Android pull-down notification menu is rather inconvenient.

So my question to @devemux86 is: is there any way to implement the disabling of the screen lock differently/in a less cumbersome way? Maybe by repeatedly tapping the same spot on the screen (which is highly unlikely to be done by raindrops), or something similar?

**Reply by devemux86** [reply]
@bios4 We can search for different "unlock" implementations.

Do other apps have this feature? How do they handle the "unlock"?

**Reply by bios4** [reply]
I don't use any other app that I know of that has this function...

Options I can think of for disabling the screen lock include:
- Multiple taps on the screen, perhaps in a specific area; maybe by displaying a "screen locked" button (lock symbol?) on the screen once the function is active, which would need to be tapped (multiple times) to deactivate it
- Use the smartphone's physical buttons (volume up/down, etc.) to disable the screen lock; however, this would be a less convenient solution for me, as the buttons could potentially be difficult to access depending on the phone's holder
- Draw a specific pattern on the screen to deactivate the function; however, I have no idea if this is even possible in Android

**Reply by devemux86** [reply]
@bios4 Note that the volume buttons can also be used to zoom the map, if enabled in map settings.

**Reply by bios4** [reply]
Right... So forget this suggestion. 😅

My favorite for sure would be the multiple-tap solution, if reasonable.
